docs/src/sdp/existing_configs.rst

```diff
@@ -177,18 +177,33 @@ Few-shot Learning Evaluation of Universal Representations of Speech (FLEURS)
 
    config-docs/armenian/fleurs/config
 
-English LibriSpeech (ELS)
-~~~~~~~~~~~~~~~~~~~~~~~~~
+LibriSpeech
+~~~~~~~~~~~
 
 **Dataset link:** https://openslr.org/12
```

Comment:
Add empty line between these 2 "dataset links" so they will appear on different lines in the docs (currently they appear on the same line)

Reply:
I think even better if:
```
**Dataset links:** https://openslr.org/12 (regular), https://openslr.org/31 (mini Librispeech)

```

Reply:
Done